diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index e9fc367..642330c 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -11,7 +11,7 @@ let isDebugMode = !module.parent;
 //for testing purpose
 if (isDebugMode) {
 
-    let PDF_FILE = './test/data/05-versions-space.pdf';
+    let PDF_FILE = './node_modules/pdf-parse/test/data/05-versions-space.pdf';
     let dataBuffer = Fs.readFileSync(PDF_FILE);
     Pdf(dataBuffer).then(function(data) {
         Fs.writeFileSync(`${PDF_FILE}.txt`, data.text, {
diff --git a/node_modules/pdf-parse/test/data/05-versions-space.pdf.txt b/node_modules/pdf-parse/test/data/05-versions-space.pdf.txt
new file mode 100644
index 0000000..a43ff7e
--- /dev/null
+++ b/node_modules/pdf-parse/test/data/05-versions-space.pdf.txt
@@ -0,0 +1,4 @@
+
+
+Dadfrtfjh,mgf  
+v.0.01 
\ No newline at end of file
